airflow/dag_processing/manager.py

```diff
@@ -216,6 +225,7 @@ def run(self):
 
         self.log.info("Getting all DAG bundles")
         self._dag_bundles = list(DagBundlesManager().get_all_dag_bundles())
+        self._symlink_latest_log_directory()
```

Comment:
@jedcunningham , This particular call happens before structlog initializes the folder. So, its always skipped here.  

One reason for this, I could think of, was that, if a dag-processor pod/process restarts/exits in between a day for some reason and then, we are rerunning it, only then this call would create symlink.

Did I guess the reason correctly or is there any other reason for  invoking `_symlink_latest_log_directory` here?